test/test-servers/koa_scaffold_server.ts

```diff
@@ -16,11 +16,12 @@
 // jscs doesn't understand koa..
 // jscs:disable
 
-var errorHandler = require('../../src/index.js')({
+import {ErrorReporting} from '../../src/index';
+const errorHandler = new ErrorReporting({
   onUncaughtException: 'report',
-});
-var koa = require('koa');
-var app = koa();
+} as {});
```

Comment:
What's with the `as {}`?

Reply:
This is needed because the constructor is using an invalid configuration option.  I think it would be better to address this in a follow-up PR.